src/main/java/com/spring/boot/repository/CommentRepository.java

```diff
@@ -9,6 +9,6 @@
 @Repository
 public interface CommentRepository extends JpaRepository<Comment, Long> {
     @Query(value = "SELECT c FROM Comment c "
-            + "WHERE c.productId = ?1")
+            + "ORDER BY c.productId DESC")
```

Comment:
if we want to get comments of the specific comment does it work ok without 'where' statement, it looks like you will fetch all the comments?

Reply:
oops...)

Reply:
agree, thank you